appveyor.yml

```diff
@@ -0,0 +1,78 @@
+image: Visual Studio 2019
```

Comment:
In WebServices I also made it so that on git tags there is a packaged zip file being created. Can you add this here too please?